frontend/plugins/vuex-persist.client.js

```diff
@@ -14,7 +14,7 @@ export default ({ store, isDev }) => {
             },
             modules: ['cookies'],
         }).plugin(store);
+        await store.dispatch('network/setup');
```

Comment:
Why did you remove `isDev` parameter, which adds extra network to our dropdown (localhost:8545) depending on the environment